packages/actions/test/e2e/01-setup.test.ts

```diff
@@ -1,58 +1,225 @@
+
+            await multiPartUpload(
+                userFunctions,
+                bucketName,
+                zkeyStorageFilePath,
+                zkeyLocalFilePath,
+                String(process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB),
+                Number(process.env.CONFIG_PRESIGNED_URL_EXPIRATION_IN_SECONDS)
+            )
+
+            // 3. upload pot
+            const potLocalFilePath = "/tmp/circuit.pot"
+            fs.writeFileSync(potLocalFilePath, "pot")
+
+            const potStorageFilePath = getPotStorageFilePath("testpot.pot")
+            await multiPartUpload(
+                userFunctions,
+                bucketName,
+                potStorageFilePath,
+                potLocalFilePath,
+                String(process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB),
+                Number(process.env.CONFIG_PRESIGNED_URL_EXPIRATION_IN_SECONDS)
+            )
+
+            // 4. upload r1cs
+            const r1csLocalFilePath = "/tmp/circuit.r1cs"
+            fs.writeFileSync(r1csLocalFilePath, "r1cs")
+
+            const r1csStorageFilePath = getR1csStorageFilePath(circuit.data.prefix!, "circuit_00001.r1cs")
+            await multiPartUpload(
+                userFunctions,
+                bucketName,
+                r1csStorageFilePath,
+                r1csLocalFilePath,
+                String(process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB),
+                Number(process.env.CONFIG_PRESIGNED_URL_EXPIRATION_IN_SECONDS)
+            )
+
+            // 5. setup ceremony
+            await setupCeremony(userFunctions, ceremony.data, ceremony.data.prefix!, [circuit.data])
+
+            // 6. confirm
+            const ceremonyDoc = await adminFirestore.collection("ceremonies").doc(ceremony.uid).get()
+            expect(ceremonyDoc.data()).to.not.be.null
+
+            // clean up
+            await deleteObjectFromS3(ceremonyBucket, zkeyStorageFilePath)
+            await deleteObjectFromS3(ceremonyBucket, potStorageFilePath)
+            await deleteObjectFromS3(ceremonyBucket, r1csStorageFilePath)
+            await deleteBucket(ceremonyBucket)
+            fs.unlinkSync(zkeyLocalFilePath)
+            fs.unlinkSync(potLocalFilePath)
+            fs.unlinkSync(r1csLocalFilePath)
+        })
+    }
+
+    it("should fail to create a new ceremony when given the wrong path to a zkey", async () => {
+        // make sure we are logged in as coordinator
+        await signInWithEmailAndPassword(userAuth, coordinatorEmail, coordinatorPwd)
+
+        // 2. multi part upload
+        assert.isRejected(
+            multiPartUpload(
+                userFunctions,
+                bucketName,
+                objectName,
+                "./nonExistantPath.zkey",
+                process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB || "128",
+                Number(process.env.CONFIG_PRESIGNED_URL_EXPIRATION_IN_SECONDS) || 3600
+            )
+        )
+    })
+
+    afterAll(async () => {
+        // Clean ceremony and user from DB.
```

Comment:
The comment says clean ceremony but I cannot see any ceremony doc deletion.

Reply:
Will be added